apidAnalytics_suite_test.go

```diff
@@ -220,7 +217,7 @@ func insertTestData(db apid.DB) {
 
 	txn, err := db.Begin()
 	Expect(err).ShouldNot(HaveOccurred())
-	txn.Exec("INSERT INTO APP_CREDENTIAL_APIPRODUCT_MAPPER (tenant_id,"+
+	txn.Exec("INSERT INTO kms_APP_CREDENTIAL_APIPRODUCT_MAPPER (tenant_id,"+
```

Comment:
These changes were made by Alex few months back so I just did the validation and fixed the test case now. Not sure if the table names are case sensitive in SQLite. 

I have tested the functionality locally and also my current code coverage is at 84% and everything is passing.